modules/ROOT/pages/dataweave-versioning-behavior.adoc

```diff
@@ -0,0 +1,17 @@
+= Versioning Behavior in DataWeave
+
+In each version of DataWeave, changes are introduced that modify the way the language behaves. When existing scripts depend on previous behavior, system properties are introduced to toggle the change so that backwards compatibilty is not broken in the current version. These system properties that control behavior in DataWeave are called *Compatibility Flags*. Each flag defined a deletion version, after which it will no longer exists, and the behavior that it controlled will default to the new one.
+
+Specifying DataWeave's *language level* indicates to the runtime which Compatibility Flags are available. As previously stated, when upgrading to a new version there are flags that will be deleted. In order to facilitate updatating DataWeave, setting the language level of a previous version makes it so that the flags available on that version are visible, mantaining backwards compatibility. Language level defaults to the running DataWeave version and has to be at most the same as the runtime version.
+
+For Mule applications, the language level is determined by the application's `minMuleVersion`. Given the `minMuleVersion`, the language level is computed by subtracting two major versions, as the following table shows:
+
+[%header%autowidth.spread,cols="a,a"]
+|===
+| minMuleVersion | Language Level
+| 4.5 | 2.5
+| 4.4 | 2.4
+| 4.3 | 2.3
+|===
+
```

Comment:
```suggestion

Setting the Mule version configures the application to use the corresponding DataWeave language level. This setting is independent of the language level set in a DataWeave script's `%dw` directive. For details, see xref:dataweave-language-introduction.adoc#dw_header[DataWeave Header] and xref:dataweave-versioning-behavior.adoc[].
```

Reply:
I would say that the setting is independent of the syntax version set in the script by the directive.

Reply:
revised, thx @andres-rad !